CONTRIBUTING.md

```diff
@@ -209,6 +209,72 @@ Before submitting your changes, run the test suite to make sure that nothing is
 ./gradlew check
 ```
 
+### Project layout
+
+This repository is split into many top level directories. The most important
+ones are:
+* docs: Documentation for the project.
+* distribution:  Builds our tar and zip archives and our rpm and deb packages.
```

Comment:
I'm on the fence with the "we"/"our" style in this case, maybe this (and the rest of this description) could be rephrased in an neutral style, here e.g. "builds the tar and zip archive ...". I haven't checked how this fits into the rest of the Contributing docs though, so please ignore if this doesn't fit.

Reply:
I tend to use we/our/you instead of the more declarative style. I figure it makes us sound like we're humans talking to humans rather than corporate robots. I'm not 100% consistent with it but I try. We have a long history of having folks write in their own voice, so you get some stuff in this style by me and others in a super formal, passive voice style.